src/utils/constants/paymentTypes.ts

```diff
@@ -0,0 +1,24 @@
+export type PaymentType = Record<string, Record< "name" | "description" | "fee", string | string |string>>;
+
+export const AccountType: PaymentType =  {
+    "CIC": { name: "CIC", description: "Community Interest Companies", fee: "£15" },
```

Comment:
These fees should be held in a config, I believe they already are there ready to go, just need to use it here.